robot/EDA/resources/EDA.py

```diff
@@ -377,24 +292,23 @@ def edit_eda_settings_checkbox(self, checkbox_label, checkbox_toggle):
             )
             self.selenium.click_element(locator_checkbox)
             self.selenium.click_element(locator_save)
-            self.selenium.capture_page_screenshot()
-            self.verify_toast_message2("Settings Saved Successfully.")
+            self.success_message("Settings Saved Successfully.")
 
     def verify_toast_message(self, value):
         """ Verifies the toast message """
         locator = eda_lex_locators["toast_message"].format(value)
         self.selenium.wait_until_page_contains_element(locator)
 
-    def verify_toast_message2(self, value):
-        """ Verifies the toast message """
-        locator = eda_lex_locators["toast_message2"].format(value)
+    def success_message(self, value):
```

Comment:
This name isn't very descriptive. When I saw it in code I assumed that it was saving a success message to the log. Instead, it is waiting for a toast message to appear. I recommend giving it a more accurate name, such as `wait_for_toast_message`